packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/DataImageViewer/DataImageViewer.stories.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import zooTheme from '@zooniverse/grommet-theme'
+import { Box, Grommet } from 'grommet'
+import { withKnobs, boolean, text, object } from '@storybook/addon-knobs'
+import { Factory } from 'rosie'
+import DataImageViewerContainer from './DataImageViewerContainer'
+import readme from './README.md'
+import backgrounds from '../../../../../../../.storybook/lib/backgrounds'
+
+const config = {
+  notes: {
+    markdown: readme
+  }
+}
+
+let zoomCallback
+
+function onZoom(type) {
+  zoomCallback(type)
+}
+
+function setZoomCallback(callback) {
+  zoomCallback = callback
+}
+
+const stories = storiesOf('Subject Viewers | DataImageViewer', module)
+
+stories.addDecorator(withKnobs)
+// stories.addParameters({ viewport: { defaultViewport: 'responsive' } })
+
+const { colors } = zooTheme.global
+
+const subject = Factory.build('subject', {
+  locations: [
+    {
+      'application/json': 'https://raw.githubusercontent.com/zooniverse/front-end-monorepo/master/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/helpers/mockLightCurves/kepler.json'
+    },
+    { 'image/png': 'https://panoptes-uploads.zooniverse.org/production/subject_location/6379335f-d893-445d-a25e-c14b83eabf63.png' }
+  ]
+})
+
+stories
+  .add('light theme', () => {
+    return (
+      <Grommet theme={zooTheme}>
+        <Box height='500px' width='700px'>
```

Comment:
The Storybook scatterplots incrementally increase in size, I think because the height is not flowing down to the `DataImageViewer` component's wrapping `<Box />` component quite right. I think the `ScatterPlotViewer` starts resizing until it reaches the maximum space it can (I think based on vx's `withParentSize`).